pkg/query/api/v1.go

```diff
@@ -288,27 +297,30 @@ func (api *API) query(r *http.Request) (interface{}, []error, *ApiError) {
 		warnmtx.Unlock()
 	}
 
+	raw := r.FormValue("query")
+	level.Debug(api.logger).Log("query", raw)
```

Comment:
I don't really like the duplication here. We log twice in error case. Fix fo this would be to have this log line at the end on success and keep those on error case on error, can we do this?

The question is do we really want to log query on every (even sucessful) request, not sure, I don't see much of benefit but we can start with that.